pkg/jira/client.go

```diff
@@ -226,7 +263,7 @@ func (c *Client) request(ctx context.Context, method, endpoint string, body []by
 
 	if c.authType == AuthTypeBearer {
 		req.Header.Add("Authorization", "Bearer "+c.token)
-	} else {
+	} else if c.authType == AuthTypeBasic {
```

Comment:
Current default auth type is basic (if nothing is set in config/env). Looks like this might break that?

Reply:
Sorry been a while - but I believe this change was made so that the login/token are provided only if authType is AuthTypeBasic. 

If authType is Bearer you add the authorization header.
If authType is Basic you add the login/token
If authType is mtls, you don't need to modify the request.

I don't believe the default auth type has changed, and will depend on how the user configured authType.